pkg/meta/model/job.go

```diff
@@ -868,6 +868,7 @@ func (job *Job) GetInvolvingSchemaInfo() []InvolvingSchemaInfo {
 // SubJob is a representation of one DDL schema change. A Job may contain zero
 // (when multi-schema change is not applicable) or more SubJobs.
 type SubJob struct {
+	Version     JobVersion      `json:"version"`
```

Comment:
can we make all job version related to multi schema change to v1 as we have discussed

Reply:
I have checked the correctness of V2 job.args. New I have roll back to V1.